google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -184,17 +212,23 @@ static IamConfiguration fromPb(Bucket.IamConfiguration iamConfiguration) {
       Bucket.IamConfiguration.UniformBucketLevelAccess uniformBucketLevelAccess =
           iamConfiguration.getUniformBucketLevelAccess();
       DateTime lockedTime = uniformBucketLevelAccess.getLockedTime();
+      String publicAccessPrevention = iamConfiguration.getPublicAccessPrevention();
 
       return newBuilder()
           .setIsUniformBucketLevelAccessEnabled(uniformBucketLevelAccess.getEnabled())
           .setUniformBucketLevelAccessLockedTime(lockedTime == null ? null : lockedTime.getValue())
+          .setPublicAccessPrevention(
+              publicAccessPrevention == null
```

Comment:
publicAccessPrevention should never be null

Reply:
Same here, if the value is never set/used then it stays null.